packages/app-project/stores/User/UserPersonalization/Notifications/helpers/getTalkUnreadConversationsIds.js

```diff
@@ -0,0 +1,31 @@
+import { talkAPI } from '@zooniverse/panoptes-js'
+
+export default async function getUnreadConversationsIds (authorization) {
+  if (!authorization) return undefined
+
+  let unreadConversationsIds = []
+
+  const getUnreadConversationsIds = async (page = 1) => {
```

Comment:
`getUnreadConversationsIds` is already declared on line 3. I’m surprised this wasn’t caught by the linter.

Reply:
Renamed getter to `getConversations`.